### PR TITLE
app-cdr/dvd+rw-tools: fix bugs

### PR DESCRIPTION
**Pull request by ceamac**
Fix problems compiling with clang.  Bump to EAPI 7.

Also, seems not maintained upstream.

### CODE REVIEW

File: app-cdr/dvd+rw-tools/dvd+rw-tools-7.1-r3.ebuild [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```
Comment (juippis) [inline app-cdr/dvd+rw-tools/dvd+rw-tools-7.1-r3.ebuild:4]: This requires a revbump and with that, resetting KEYWORDS.

File: app-cdr/dvd+rw-tools/dvd+rw-tools-7.1-r3.ebuild [hunk 2]
```diff
@@ -28,8 +28,11 @@ PATCHES=(
 	"${FILESDIR}"/${PN}-7.1-lastshort.patch
 	"${FILESDIR}"/${PN}-7.1-bluray_srm+pow.patch
 	"${FILESDIR}"/${PN}-7.1-bluray_pow_freespace.patch
+	"${FILESDIR}"/${PN}-7.1-clang.patch
 )
 
+DOCS=( index.html )
```
Comment (juippis) [inline app-cdr/dvd+rw-tools/dvd+rw-tools-7.1-r3.ebuild:34]: Are there no other docs included? Wouldn't this be better to be installed as `HTML_DOCS`?

File: app-cdr/dvd+rw-tools/files/dvd+rw-tools-7.1-clang.patch [hunk 3]
```diff
@@ -0,0 +1,71 @@
+--- a/dvd+rw-booktype.cpp	2006-06-24 12:21:48.000000000 +0300
++++ b/dvd+rw-booktype.cpp	2021-03-14 10:16:02.533741829 +0200
```
Comment (juippis) [inline app-cdr/dvd+rw-tools/files/dvd+rw-tools-7.1-clang.patch:2]: Did you find this somewhere? We prefer to keep headers in patch files for traceability.

Comment (ceamac) [reply app-cdr/dvd+rw-tools/files/dvd+rw-tools-7.1-clang.patch:2]: No, I tried to compile it with clang and fixed the bugs.  There is a patch attached to one of the bugs, but it is not complete for clang 11.

File: app-cdr/dvd+rw-tools/dvd+rw-tools-7.1-r3.ebuild (hunk 2 above)
Comment (ceamac) [reply app-cdr/dvd+rw-tools/dvd+rw-tools-7.1-r3.ebuild:34]: This is the only doc.  And it's a text file, not html.  Just has a weird name

**Comment by ceamac** [comment]
I added 7.1-r4 and set all keywords to unstable.  Please review

File: app-cdr/dvd+rw-tools/dvd+rw-tools-7.1-r4.ebuild [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+	"${FILESDIR}"/${PN}-7.1-bluray_pow_freespace.patch
+	"${FILESDIR}"/${PN}-7.1-clang.patch
+)
+
+# this is a text file, not html
+DOCS=( index.html )
+
+src_prepare() {
+	# Linux compiler flags only include -O2 and are incremental.
+	sed -i '/FLAGS/s:-O2::' Makefile.m4 || die "failed to sed out FLAGS"
```
Comment (juippis) [inline app-cdr/dvd+rw-tools/dvd+rw-tools-7.1-r4.ebuild:39]: This delimiter has a tendency of breaking gcc, 
https://bugs.gentoo.org/685160

will fix it but just FYI.

**Review by juippis — APPROVED** [review]

File: app-cdr/dvd+rw-tools/dvd+rw-tools-7.1-r4.ebuild (hunk 4 above)
Comment (juippis) [reply app-cdr/dvd+rw-tools/dvd+rw-tools-7.1-r4.ebuild:39]: Wait, there's nothing wrong with it. Sorry...

**Pull request closed without merging**